Sources/Rendering/Core/ColorTransferFunction/index.js

```diff
@@ -214,22 +214,29 @@ function vtkColorTransferFunction(publicAPI, model) {
   // Set nodes directly
   publicAPI.setNodes = (nodes) => {
     if (model.nodes !== nodes) {
+      const before = JSON.stringify(model.nodes);
       model.nodes = nodes;
-      publicAPI.sortAndUpdateRange();
+      const after = JSON.stringify(model.nodes);
+      return publicAPI.sortAndUpdateRange() || before !== after;
     }
+    return false;
   };
 
   //----------------------------------------------------------------------------
   // Sort the vector in increasing order, then fill in
   // the Range
```

Comment:
you might want to update the documentation to explain what is returned

Reply:
done (did not create a release for it) but one will be available soon with the speed of vtk.js releases. ;-)